src/zope/app/apidoc/codemodule/browser/README.rst

```diff
@@ -44,7 +44,7 @@ Return info objects for all classes in this module.
   >>> pprint(details.getClasses())
   [{'doc': 'Represent the code browser documentation root',
     'name': 'CodeModule',
-    'url': 'http://127.0.0.1/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
+    'url': 'http://127.0.0.1/++etc++site/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
```

Comment:
I *thought* it was due to a different `rootFolder` setup caused by differences in the way `zope.app.testing.setup.PlacefulSetup` and `zope.app.component.testing.PlacefulSetup` create things. But now I'm not entirely sure.

The `++apidoc++` namespace *always* reparents utilities it looks up to be beneath itself, which doesn't exactly seem right (especially for non-persistent utilities, because they're shared in memory, so we have a thread error). If I make it stop doing that, these all change back to remove the `++etc++site`...but then I run into different test isolation issues.